### PR TITLE
feat: add radius design tokens

### PR DESCRIPTION
**Pull request by SimonLeclercq**
Basic set for design tokens `radius`, with their description.

Related to #4.

### CODE REVIEW

File: design-tokens/Decathlon-Origins-Light.json [hunk 1]
```diff
@@ -928,5 +928,43 @@
...
+  "BorderRadius": {
+    "2xs": {
+      "value": "{BorderRadius.0}",
+      "type": "borderRadius"
+    },
+    "xs": {
+      "value": "{BorderRadius.100}",
+      "type": "borderRadius"
+    },
+    "m": {
```
Comment (lauthieb) [inline design-tokens/Decathlon-Origins-Light.json:941]: Where is the `"s"`?

File: design-tokens/Base.json [hunk 2]
```diff
@@ -437,5 +437,52 @@
...
+      "value": "0px",
+      "type": "borderRadius",
+      "description": "Sets a 0px corner-radius"
+    },
+    "100": {
+      "value": "4px",
+      "type": "borderRadius",
+      "description": "Sets a 4px corner-radius"
+    },
+    "200": {
```
Comment (lauthieb) [inline design-tokens/Base.json:452]: Why do you use 100, 200, 300 and no calculation and in spacings PR #98 you use a key named as the multiplier of your basis?

**Review by lauthieb — CHANGES_REQUESTED** [review]

**Comment by SimonLeclercq** [comment]
Numerical scale is now used for border-radius

**Review by lauthieb — APPROVED** [review]

File: design-tokens/Decathlon-Origins-Light.json [hunk 3]
```diff
@@ -928,5 +928,43 @@
       "value": "{TextCase.Uppercase}",
       "type": "textCase"
     }
+  },
+  "BorderRadius": {
+    "xs": {
+      "value": "{BorderRadius.0}",
+      "type": "borderRadius"
+    },
```
Comment (thibault-mahe) [inline design-tokens/Decathlon-Origins-Light.json:936]: Does it mean that when you set the size of your border-radius to xs using the `borderRadius-xs` token, actually it doesn't set any border-radius? It seems a bit counter-intuitive to me 😬

Comment (lauthieb) [reply design-tokens/Decathlon-Origins-Light.json:936]: +1

**Pull request merged**